packages/inter-protocol/src/clientSupport.js

```diff
@@ -394,3 +391,17 @@ export const Offers = {
     AddCollateral: makeAddCollateralOffer,
   },
 };
+
+/**
+ *
+ * @param {import('@agoric/vats/tools/board-utils.js').AgoricNamesRemotes} agoricNames
```

Comment:
I wonder about narrowing the type:

```suggestion
 * @param {Pick<import('@agoric/vats/tools/board-utils.js').AgoricNamesRemotes, 'brand' | 'vbankAsset'>} agoricNames
```